scrunch/datasets.py

```diff
@@ -2495,3 +2493,16 @@ def transformations(self):
         if 'transform' in self.view:
             return self.view.transform.insertions
         return None
+
+    def rollup_resolution(self, resolution):
+        """
+        PATCHes the rollup_resolution attribute of a datetime variable. This is the
+        equivalent to the UI's change resolution action.
+        """
+        assert self.type == 'datetime', 'Method only allowed for datetime variables'
+        self.dataset._validate_vartypes(self.type, resolution=resolution)
+
+        payload = json.dumps({'rollup_resolution': resolution})
+        resp = self.resource.patch(payload)
+        self.dataset._reload_variables()
```

Comment:
If you have `self.resource` to be the Variable shoji entity, you only need to do `self.resource.edit(resolution=new_resolution)` no need to do json.dumps.

Reply:
yes that the original intention but since it was failing I decided to test other paths. Neither of the methods works for this attribute.